web/src/lib/components/faces-page/assign-face-side-panel.svelte

```diff
@@ -128,7 +128,7 @@
       searchedPeopleCopy = data;
       searchWord = searchName;
     } catch (error) {
-      handleError(error, "Can't search people");
+      await handleError(error, "Can't search people");
```

Comment:
Would it make sense to just not await all of these, but change `handleError` to return void instead?